src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,121 @@
+
+<figure role="figure" aria-label="Add image descriptions (alt text) to images in documents, as in Microsoft Word's Alt Text option.">
+	<img alt="Word document has photo inserted and selected. In the right pane, the Alt Text option is open and showing an input field that reads 'White and yellow flowers'." src="/img/posts/2021-06-14-Everyday_Accessibility/Windows-AltText.png" />
+	<figcaption>Add image descriptions ("alt text") to images in documents, as in Microsoft Word's Alt Text option.</figcaption>
+</figure>
+
+### Use accessibility checkers
+Microsoft products (namely Word and PowerPoint) come with a “Check Accessibility” option. Use this to double-check you’ve addressed key accessibility needs.
+
+<figure role="figure" aria-label="Microsoft's Check Accessibility feature helps ensure your document is inclusive.">
+	<img alt="The Check Accessibility menu option is activated on a simple PowerPoint deck where one slide is missing a title. The Accessibility pane shows the 'Missing slide title' error and the mouse hover text for that slide also reads '[No Title]'." src="/img/posts/2021-06-14-Everyday_Accessibility/PPT_A11yChecker.png" />
+	<figcaption>Microsoft's "Check Accessibility" feature helps ensure your document is inclusive.</figcaption>
+</figure>
+
+## Audio and Video Content
+
+### Have clear audio
+Though all of our devices can record audio and video on-demand, sound quality can vary.  Be sure to speak as clearly as possible, preferably at a steady pace and reasonable volume, and remove or avoid background noise including echo, empty room effect, humming, or white noise.
+
+> Using a microphone (even a common corded headset) can help amplify voices and minimize distracting unwanted sounds.
+
+### Reduce background music
+Instrumental music can enhance content’s quality, but not when it reduces the ability to hear the main speaker. When using background music, keep it to a minimum (such as during an intro) or, if used throughout, ensure it is very low.
+
+> Consider putting your background music at the volume of a whisper.
+
+### Provide text with multimedia
+As with images, video and audio can be engaging but also inaccessible without a text equivalent for people who are deaf-blind, having sensory processing disorders, or have hearing impairments. Having media content written out as text is vital for inclusiveness. Text equivalents can include transcripts that type out spoken dialogue, or an article that accompanies a video (as in a video that supplements a news article).
+
+> Where there’s media, make sure there’s also text.
+
+### Do not have any flashing content
+Content that flashes on the screen more than 3 times per second is dangerous as it can trigger a seizure in some viewers. Use rapid movement sparingly in your videos and ensure nothing flashes.
```

Comment:
```suggestion
Content that flashes on the screen more than [3 times per second](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-three-times.html) is dangerous as it can trigger a seizure in some viewers. Use rapid movement sparingly in your videos and ensure nothing flashes.
```

You don't have to accept this suggestion, but I oftentimes find myself needing to invoke it for arguments involving animation. People get emotional about removing animated content, even if it's dangerous, so it might be helpful for the reader to have a resource to use if things get dicey.

Reply:
Yikes! Okay, let me see if I can rephrase to have a better link text and I'll also point to v2.1 just so folks are dropped into the latest guidelines.